### ISSUE TITLE
Remove IE from BCD

### ISSUE THREAD

**Issue by queengooborg** [mdn/browser-compat-data#15333]
Microsoft has announced an official End of Life date for Internet Explorer to be June 15, 2022[^1][^2].  In the documentation, Microsoft recommends Edge's built-in "IE mode" for viewing legacy sites which require IE.  To add to it, Internet Explorer has 0.47% of the global market share now, which is less than UC Browser's (which we removed a while back)[^3].

We should consider removing IE sometime soon due to these combined factors, most likely on or around the EOL date.

[^1]: https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support#:~:text=Internet%20Explorer%20(IE)%2011%20is,certain%20versions%20of%20Windows%2010*.
[^2]: https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/
[^3]: https://gs.statcounter.com/

**Comment by Seirdy** [comment]
IE still has a high market share in regions like Japan (~5%, give or take a few, depending on your source).

It might be worth replacing IE with "Edge IE Mode" or similar, since that will be maintained for the next several years.

**Comment by makotokato** [comment]
IE11 for Windows 10 will be EOL, but IE11 on Windows server etc aren't EOL. See https://techcommunity.microsoft.com/t5/windows-it-pro-blog/internet-explorer-11-desktop-app-retirement-faq/ba-p/2366549

**Comment by Seirdy** [comment]
Another data point: 3.3% of respondents to the latest WebAIM screenreader survey use IE: https://webaim.org/projects/screenreadersurvey9/#browsers

IE has long had disproportionate representation among screenreader users. Combine this with the fact that Edge's "IE Mode" will be supported for several years and it looks like having some docs about its feature support would be helpful.

**Comment by ddbeck** [comment]
Some related discussion on mdn/content: https://github.com/mdn/content/discussions/14953. One thing I'd highlight from my comment there: there will be a period when IE information will continue to be useful, even after EOL (e.g., for developers migrating applications near or shortly past the mainstream EOL, or developers in environments with extended support).

I'd suggest adopting a posture of conservatism when it comes to removing such data from BCD itself. In line with other irrelevance guidelines, 2 years after EOL would be conventional, though it might make sense to drop it sooner, if, for example, BrowserStack and similar tools dropped IE.

**Comment by lukewarlow** [comment]
If it's not possible to remove all support data would it be possible to treat it more like node and deno where we only include it if it has support? This would keep the data for existing APIs, but would mean new APIs that IE will never support don't need to include the false data, and if I understand the MDN consumer properly wouldn't show in the MDN browser table.

**Comment by jamiebuilds** [comment]
Raising this again now that June 15th has come and gone. 

1. Would it be possible to run an experiment where IE data is hidden behind a button and MDN measures how many people actually click the button?
2. Should there be a separate section for deprecated browsers? Does the MDN team really want to maintain that forever (keeping in mind that "maintaining" it is just saying "not supported" for every new feature from now on)

**Comment by NagayamaToshiaki** [comment]
@Seirdy In this logic, we need to add information about Netscape compatibility. Legacys should be buried and forget them (unless you're a historian).

**Comment by teoli2003** [comment]
Closing this. We are well aware of the IE status, and we will not forget about it. No need to keep this duplicate open.

**Comment by queengooborg** [comment]
TLDR for everyone regarding IE data: after valuable feedback and some careful deliberation, we have decided that removing IE from BCD itself is a little too early, however we are interested in experimenting around with the rendering on MDN Web Docs to hide IE.  While the EOL of IE for desktop consumers has finally come, IE is still under support in certain circumstances (and still heavily used in certain markets).  We'll re-evaluate removal in two years or so!

Since IE is dead, however, I will personally not be focusing on maintaining IE data.  While I won't turn down others' PRs to fix IE data, I will not be writing any myself (or writing few that have changes for other browsers), and will consider IE-related issues to be low priority.

**Comment by carlosjeurissen** [comment]
Even tho this issue is closed, I still want to at the following: it may not be that simple, yet an out of the box solution might be to just regard IE as being the same browser as Edge. IE stops at version 11, and Edge starts at version 12. Just like Opera and Edges switch to the Chromium engine, which also did not mean a split in the browser compat data.

We can come up with an algorithm which merges the compat data from IE and Edge. If incompatible with IE and compatible with edge. we can state compatibility starting Edge v12+ if edge includes the feature from day 1. If something was already supported in IE8, we can say edge starting from version 8. For added clarity we can rename "edge" to "IE/Edge".
